src/main/java/entities/Game.java

```diff
@@ -121,6 +131,14 @@ public Deck getGameDeck() {
         return this.gameDeck;
     }
 
+    /**
+     * Get the top Card of this Game's Deck.
+     * @return A Card object representing the top Card of this Game's Deck.
+     */
+    public Card getTopCard() {
```

Comment:
Should this method set currentDrawnCard to the Card returned by this method?

Reply:
good catch